lib/node/nodes/trade-area.js

```diff
@@ -30,9 +30,9 @@ TradeArea.prototype.sql = function() {
     var template = this.dissolved ? tradeAreasDissolvedQueryTemplate : tradeAreasQueryTemplate;
     const columnsQuery = this.source.getColumns(true)
         .filter(columnName => !(DROP_COLUMN[columnName]))
-        /*jshint unused: false */
-        .map(columnName => `'{$columnName}'`)
-        /*jshint unused: true */
+        /*jshint quotmark: false */
+        .map(columnName => "\"" + columnName + "\"")
```

Comment:
what abaut \`"{$columnName}"\` ?

Reply:
It didn't work because I don't know who write string templates properly:
``` 
`"{$columnName}"` 
````

vs

``` 
`"${columnName}"` 
````